plugin-template

```diff
@@ -98,6 +98,9 @@ DEPRECATED_FILES = {
         "docs/_static/.this_makes_it_so_git_keeps_the__static_folder",
         "docs/_static/api.json",
     ],
+    "bootstrap": [
+        "flake8.cfg",
+    ],
```

Comment:
Major problem: flake8.cfg was seeded as part of bootstrap, so adding this change will break a plugins CI unless flake8.cfg is renamed to .flake8, but that won't be warned for by `./plugin-template --github`...